src/test/codegen/repeat-trusted-len.rs

```diff
@@ -14,6 +14,6 @@ pub fn helper(_: usize) {
 // CHECK-LABEL: @repeat_take_collect
 #[no_mangle]
 pub fn repeat_take_collect() -> Vec<u8> {
-// CHECK: call void @llvm.memset.p0i8.[[USIZE]](i8* {{(nonnull )?}}align 1 %{{[0-9]+}}, i8 42, [[USIZE]] 100000, i1 false)
```

Comment:
Is this regressing and now the buffer size given to the intrinsic is not constant anymore?

Reply:
Not exactly. IDK why but we get a single `store` and then `memset` with size = 99999 here.

Reply:
Fair, can the test check for this specific pattern please? With `{{.*}}` it is possible to regress to, say, 100000 memsets all with 1 byte size without noticing.

Reply:
Done.